code/game/objects/items/devices/radio/headset_vr.dm

```diff
@@ -91,15 +91,15 @@
 	icon_state = "pilot_headset_alt"
 
 /obj/item/device/radio/headset/explorer
-	name = "explorer's headset"
-	desc = "Headset used by explorers for exploring. Access to the explorer channel."
+	name = "away team's headset"
```

Comment:
away team member's maybe? Its minor, but wording of 'away team's headset' feels a tad off

Reply:
Volunteer's headset maybe?